thinkflow/migrations/0005_plantdisease_input_tokens_plantdisease_output_tokens_and_more.py

```
@@ -0,0 +1,28 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('thinkflow', '0004_remove_plantdisease_details'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='plantdisease',
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)